BUILDING.md

```diff
@@ -103,6 +103,7 @@ Prerequisites:
 * Basic Unix tools required for some tests,
   [Git for Windows](http://git-scm.com/download/win) includes Git Bash
   and tools which can be included in the global `PATH`.
+* vcbuild is provided by this repo as a batch file so if using powershell, use local syntax `./vcbuild nosign`
```

Comment:
Since this is Windows, wouldn't `.\` be more correct (and work in `cmd.exe` and PowerShell)? Is there any reason not to prefix all the examples `.\` (e.g. `.\vcbuild`) to be consistent with `./configure`/`./node` examples in the Unix / OS X section?